provider/provider.go

```diff
@@ -897,6 +932,11 @@ func resourceShorelineObject(configJsStr string, key string) *schema.Resource {
 		sch.Required = GetNestedValueOrDefault(attrMap, ToKeyPath("required"), false).(bool)
 		sch.Computed = GetNestedValueOrDefault(attrMap, ToKeyPath("computed"), false).(bool)
 		sch.ForceNew = GetNestedValueOrDefault(attrMap, ToKeyPath("forcenew"), false).(bool)
+		sch.Deprecated = GetNestedValueOrDefault(attrMap, ToKeyPath("deprecated"), "").(string)
+		conflictsWith, ok := GetNestedValue(attrMap, ToKeyPath("conflicts_with"))
```

Comment:
Should this not also include `alias` and `prefrered_alias` entries?

Reply:
As alert_manager support was targeted for astronomer but they haven't signed, this has been put on hold. We've only added the reserved words (`external_trigger_id` and `trigger_source`) commented out.